roottest/root/io/transient/base/execWriteFile.ref

```diff
@@ -1,5 +1,4 @@
 
-(int) 0

```

Comment:
I am vaguely surprised those disappeared ... (this seems related to the comment at line 7 of roottest/root/io/transient/base/CMakeLists.txt)

Reply:
Ah. I see.  I had missed the additional `(void)`.  Thanks.